src/generator68k/Makefile.am

```diff
@@ -1,43 +1,29 @@
 ## Process this file with automake to produce Makefile.in
 
 noinst_PROGRAMS = def68k$(EXEEXT) gen68k$(EXEEXT)
-BUILT_SOURCES = def68k-iibs.h def68k-funcs.h def68k-proto.h \
-                cpu68k-0.c cpu68k-1.c cpu68k-2.c cpu68k-3.c \
-                cpu68k-4.c cpu68k-5.c cpu68k-6.c cpu68k-7.c \
-                cpu68k-8.c cpu68k-9.c cpu68k-a.c cpu68k-b.c \
-                cpu68k-c.c cpu68k-d.c cpu68k-e.c cpu68k-f.c
 
 ## stage one - create iibs, funcs and proto header files from def68k.def
 
-#def68k_SOURCES = def68k.c tab68k.c
 AM_CPPFLAGS = -I../ -I.
 
-#head_SOURCES = def68k-iibs.h def68k-funcs.h def68k-proto.h
-
-## see automake.info section on 'Support for executable extensions'
-
-#head$(EXEEXT):	def68k-iibs.h def68k-funcs.h def68k-proto.h
-
-#def68k-iibs.h:	def68k def68k.def
-#	./def68k
-#def68k-funcs.h:	def68k def68k.def
-#	./def68k
-#def68k-proto.h:	def68k def68k.def
-#	./def68k
-
 gcc_build_verbose = $(gcc_build_verbose_$(V))
 gcc_build_verbose_ = $(gcc_build_verbose_$(AM_DEFAULT_VERBOSITY))
 gcc_build_verbose_0 = @echo "  HCC    $@";
 
-def68k-proto.h def68k-funcs.h def68k-iibs.h : def68k$(EXEEXT) def68k.def
+def68k_SOURCES = def68k.c tab68k2.c
+
+def68k.stamp: def68k$(EXEEXT) def68k.def
 	$(AM_V_GEN)./def68k$(EXEEXT) 2>&1 >/dev/null
+	touch def68k.stamp
```

Comment:
Can you make it a single line with e.g. "<cmd1> && <cmd2>" to not run touch if def68k failed?